requirements.txt

```diff
@@ -2,11 +2,11 @@ aiohttp==3.9.4
 attrs==21.4.0
 azure-identity==1.12.0
 azure-mgmt-billing==6.0.0
-azure-mgmt-resource==23.0.1
 azure-mgmt-compute==30.1.0
+azure-mgmt-costmanagement==3.0.0
 azure-mgmt-monitor==6.0.2
 azure-mgmt-network==25.0.0
-azure-mgmt-costmanagement==3.0.0
+azure-mgmt-resource==23.0.1
```

Comment:
Did u update the related code ?

Reply:
pre-commit sorted the requirements.txt file